funcake_dags/scripts/transform.sh

```diff
@@ -21,6 +21,10 @@ XSL=https://raw.githubusercontent.com/${XSL_REPO}/${XSL_BRANCH}/${XSL_FILENAME}
 echo Transformation File: $XSL
 
 # Grab list of items from designated aws bucket (creds are envvars), then index each item
+if [ -z ${XSL_FILENAME} ] || [ ${XSL_FILENAME} == "transforms/" ]; then
+	echo "There is no XSL_FILENAME defined."
+	exit 1
+else
```

Comment:
@sensei100 there's no reason to make this an if/else because we are exiting the program after "if" evaluates.

Reply:
@dkinzer Okay, I took it out.